woocommerce/wc-admin
# Add settings client pages and plumbing

Adds the pages to the client-side router and sets up the client-side render.

### Screenshots
<img width="475" alt="Screen Shot 2021-01-18 at 7 38 35 PM" src="https://user-images.githubusercontent.com/10561050/104974178-00190080-59c5-11eb-90f8-3ca687a712f7.png">


### Detailed test instructions:

1. Enable the new settings feature.
2. (optional) Enable the new nav feature to allow easier navigation.
3. Click around to the various settings pages to make sure the settings render (currently breadcrumbs should show the setting name, but the page will simply show `Settings page`).
